### PR TITLE
Fix issue 14301

### PR DESCRIPTION
**Pull request by ZhangRuiMingZRM**
Hello!

* Type: bug fix 
* Link to issue: https://github.com/phalcon/cphalcon/issues/14301

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:

Thanks

### CODE REVIEW

**Comment by Jurigag** [comment]
We don't accept PRs to master, can you send it to proper branch(4.0.x being current developement branch).

**Comment by ZhangRuiMingZRM** [comment]
> We don't accept PRs to master, can you send it to proper branch(4.0.x being current developement branch).



> We don't accept PRs to master, can you send it to proper branch(4.0.x being current developement branch).

Ok :)

**Comment by sergeyklay** [comment]
@ZhangRuiMingZRM Could you please update change log?

**Review by sergeyklay — DISMISSED** [review]

File: phalcon/Helper/Str.zep [hunk 1]
```diff
@@ -361,9 +361,9 @@ class Str
     final public static function includes(string! needle, string! haystack) -> bool
     {
         if function_exists("mb_strpos") {
-            return (bool) mb_strpos(haystack, needle);
+            return (bool) false !== mb_strpos(haystack, needle);
         } else {
-            return (bool) strpos(haystack, needle);
+            return (bool) false !== strpos(haystack, needle);
```
Comment (Jeckerson) [inline phalcon/Helper/Str.zep]: Why cast to `bool` if proper condition always returns `bool`?

Comment (niden) [reply phalcon/Helper/Str.zep]: I think that was remnant of the old code. Removing the `(bool)` will do what we need.

Comment (ZhangRuiMingZRM) [reply phalcon/Helper/Str.zep]: Ok, I'll do that on Monday

**Comment by ZhangRuiMingZRM** [comment]
@niden  Please review

**Comment by CameronHall** [comment]
> @niden Please review

@ZhangRuiMingZRM can you please address Jeckerson's review. It's about removing the cast to bool on the return statements.

**Comment by ZhangRuiMingZRM** [comment]
> > @niden Please review
> 
> @ZhangRuiMingZRM can you please address Jeckerson's review. It's about removing the cast to bool on the return statements.

File: CHANGELOG-4.0.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 - `Phalcon\Mvc\Collection::cancelOperation()` now returns `true` on success.
 - Fixed `Phalcon\Application\AbstractApplication` interfaces for `registerModules()`, `setDefaultModule()` and `setEventsManager()` [#14293](https://github.com/phalcon/cphalcon/issues/14293)
 - Fixed `Phalcon\Paginator\Adapter\QueryBuilder` to throw exception on incorrect parameter [#14299](https://github.com/phalcon/cphalcon/issues/14299)
+- Fixed `Phalcon\Helper\Str::includes()` to return correct result [#14301](https://github.com/phalcon/cphalcon/issues/14301)
```
Comment (ruudboon) [inline CHANGELOG-4.0.md]: Can you move this up to the upcoming release? beta.2 already dropped

**Review by ruudboon — CHANGES_REQUESTED** [review]
Code looks good to me. Please update the CHANGELOG so we can merge this. Thnx

**Review by sergeyklay — APPROVED** [review]

**Review by ruudboon — APPROVED** [review]

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**